wagtail_extensions/models.py

```diff
@@ -28,6 +28,7 @@ class Meta:
     body = fields.StreamField([
         ('text', extension_blocks.TextBlock()),
         ('table', TableBlock()),
+        ('images', extension_blocks.ImagesBlock()),
```

Comment:
This is a bit weird - you have a StructBlock whose only child is a ListBlock. Better just to make ImagesBlock a ListBlock itself?